corehq/apps/reports/standard/users/reports.py

```diff
@@ -40,7 +42,7 @@
 class UserHistoryReport(GetParamsMixin, DatespanMixin, GenericTabularReport, ProjectReport, PaginatedReportMixin):
     slug = 'user_history'
     name = gettext_lazy("User History")
-    section_name = gettext_lazy("User Management")
+    section_name = gettext_lazy("Users")
```

Comment:
As much as I hate that this is done through inheritance, should this change be effected by extending `BaseUserSettingsView`?
https://github.com/dimagi/commcare-hq/blob/4c8bbe4093fc3501899480813ead8cb58c2222dd/corehq/apps/users/views/__init__.py#L143-L143

I don't particularly object to the repetition if that interferes with the other *five* (:eyes:) ancestor classes.

Reply:
I think it'd be a little misleading, because this page wouldn't use the `section_url` that's defined in `BaseUserSettingView`, since reports make their own breadcrumbs: https://github.com/dimagi/commcare-hq/blob/4bc0c8a896f82a3edc96824ed10eb7f67e9925d8/corehq/apps/reports/templates/reports/standard/base_template.html#L60

Renaming `default_report_url` to `section_url` to make these two areas more compatible might not be a bad idea, though I'm not sure I want to add it to this PR.